src/components/PlayerModal.jsx

```diff
@@ -18,10 +18,36 @@ import {
 	calculateOPS
 } from '../utils/hittingStatsCalculations'
 import { calculateEarnedRunAverage } from '../utils/pitchingStatsCaclulations'
+import pitchingStatsConfig from '../utils/pitchingStatsConfig'
+import hittingStatsConfig from '../utils/hittingStatsConfig'
 
 const PlayerModal = ({ setPlayerData, handlePlayerModalClose }) => {
 	const playerData = setPlayerData
 
+	const getStatValue = (playerData, key) => {
```

Comment:
see comment on the config file, suggested for every stat, it should have a designated callback, and that's what you pass in to the component instead. For all other stats that will only need to be reduced, you have to create that function and add that in instead.